.ruby-version

```diff
@@ -1 +1 @@
-2.7.6
+3.2.0
```

Comment:
let me know if you want me to remove this from the PR 😅

Reply:
Probably, but this is just the default version being used, it has little consequence on anything. We should probably just delete this file.